packages/quantic/cypress/integration/standalone-search-box/standalone-search-box-actions.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  StandaloneSearchBoxSelector,
+  StandaloneSearchBoxSelectors,
+} from './standalone-search-box-selectors';
+
+const standaloneSearchBoxActions = (selector: StandaloneSearchBoxSelector) => {
+  return {
+    typeInSearchBox: (query: string) => {
+      selector
+        .input()
+        .invoke('val', query)
+        .logAction(`when typing "${query}" in search box`);
+    },
+    submitSearch: () => {
+      selector.searchButton().click().logAction('when submitting query');
+    },
+    focusSearchBox: () => {
+      selector.input().trigger('keydown', {keyCode: 27});
+      cy.wait(500);
+      selector.input().trigger('keyup', {keyCode: 27});
```

Comment:
Why not use `selector.input().focus()` instead?

Please use a constant for `keyCode: 27`, I didn't remember it was the escape key :) Is it mandatory to use the escape key, or any other key could do?

Also 500 ms is a pretty long delay between `keydown` and `keyup` events. Is this delay even required?

Reply:
I tried `selector.input().focus()` but it didn't work and the test was too flaky
I found that simulating the whole action of escape key works better

Reply:
Ok, thanks!